dcos/subcommand.py

```diff
@@ -14,7 +14,7 @@
 def command_executables(subcommand, dcos_path):
     """List the real path to executable dcos program for specified subcommand.
 
-    :param subcommand: name of subcommand. E.g. marathon
+    :param subcommand: name of subcommand. E.g. Marathon
```

Comment:
The example should be in lower case, no? The package name is `marathon` not `Marathon` I think.